bin/publish_private_pod

```diff
@@ -1,10 +1,15 @@
 #!/bin/bash -eu
 
 PATCH_COCOAPODS="false"
+COCOAPODS_FLAGS=(--verbose)
 
 while [[ "$#" -gt 0 ]]; do
 	case $1 in
-		--patch-cocoapods) PATCH_COCOAPODS="true" ;;
+		--patch-cocoapods)
+			PATCH_COCOAPODS="true" ;;
```

Comment:
For stylistic consistency with rest of the code:
```suggestion
			PATCH_COCOAPODS="true"
			;;
```

Reply:
Good catch! Updated in https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/pull/86/commits/a8e3224672945c82b326033156c577077ba5816d.

Reply:
Mmh seems that commit mixed tabs on line 9 vs spaces in line 10, which made the statements indentation misaligned (depending on your tab width settings)